packages/utils/telemetry-utils/src/eventEmitterWithErrorHandling.ts

```diff
@@ -11,18 +11,30 @@ import { TypedEventEmitter, EventEmitterEventType } from "@fluidframework/common
  * Any exception thrown by "error" listeners will propagate to the caller.
  */
 export class EventEmitterWithErrorHandling<TEvent extends IEvent = IEvent> extends TypedEventEmitter<TEvent> {
+    constructor(private readonly errorListener?: (eventName: EventEmitterEventType, error: any) => void) {
```

Comment:
I'd call it errorHandler, since nothing else happens to the error (errorListener sounds like it will also go through the default path)

Reply:
And what do you think about having it return a bool and returning that for `emit`? Maybe add that later if/when it's needed and just stick with return `true` for now - is that what you're thinking?

Reply:
Changing name. There is no need to return a value here, as it's either handled, or exception is thrown - there is no path where false is possible today (as return value indicates whether any handlers were called)